test/box/graceful_shutdown.lua

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env tarantool
+local os = require('os')
+
+box.cfg{
+    listen              = os.getenv("LISTEN"),
+    memtx_memory        = 107374182,
+    pid_file            = "tarantool.pid",
+    iproto_threads      = 10,
+    wal_max_size        = 2500
```

Comment:
This value is not used in this test as i think

Reply:
Removed unnecessary options.